interfaces/builtin/spi.go

```diff
@@ -55,7 +55,7 @@ func (iface *spiInterface) StaticInfo() interfaces.StaticInfo {
 	}
 }
 
-var spiDevPattern = regexp.MustCompile("^/dev/spidev[0-9].[0-9]+$")
+var spiDevPattern = regexp.MustCompile("^/dev/spidev[0-9]+.[0-9]+$")
```

Comment:
AIUI the period is a literal. As such, can you escape it in the regexp, please? I.e. something like

    `^/dev/spidev[0-9]+\.[0-9]+$`

(note the backticks to avoid having to double up the backslash).

Reply:
@chipaca thanks for spotting this!
PR is now updated